pkgs/desktops/gnome/extensions/hanabi/default.nix

```diff
@@ -0,0 +1,89 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  appstream-glib,
+  clapper,
+  gettext,
+  gjs,
+  glib,
+  gobject-introspection,
+  gst_all_1,
+  gtk4,
+  libadwaita,
+  makeWrapper,
+  meson,
+  ninja,
+  nodePackages,
+  shared-mime-info,
+  wrapGAppsHook4,
+}:
+
+stdenv.mkDerivation (finalAttrs: rec {
+  pname = "gnome-shell-extension-hanabi";
+  version = "0-unstable-2024-12-01";
+
+  src = fetchFromGitHub {
+    owner = "jeffshee";
+    repo = "gnome-ext-hanabi";
+    rev = "3ed1e3c4e6a533d3f8e5bf38fbf994e6d134f473";
+    hash = "sha256-6oW4d2uHl2QfMlZBC2Ih18Hoot3j2g5y0NNc8TCHVpY=";
+  };
+
+  passthru = {
+    extensionUuid = "hanabi-extension@jeffshee.github.io";
+    extensionPortalSlug = "hanabi";
+  };
+
+  dontBuild = false;
+  dontWrapGApps = true;
+
+  nativeBuildInputs = [
+    appstream-glib
```

Comment:
Where is this used? 

We usually skip appstream file validation since it is unlikely to reveal any issues. Also `appstream-glib` is in a heavy maintenance mode (projects should migrate to `appstream`).